Vagrantfile

```diff
@@ -20,37 +20,27 @@ fi
 PROXYSCRIPT
 
 $build_and_test = <<SCRIPT
+# fix "dpkg-reconfigure: unable to re-open stdin: No file or directory" issue
 export DEBIAN_FRONTEND=noninteractive
 
 echo "Provisioning system to compile, test and develop."
-# fix "dpkg-reconfigure: unable to re-open stdin: No file or directory" issue
-sudo dpkg-reconfigure locales
-
 sudo apt-get update -qq -y
 
-sudo apt-get install -qq curl php5 git libspreadsheet-writeexcel-perl libdbd-sqlite3-perl
-
-# sudo /vagrant/install/install_composer.sh
-
-# install spdx-tools
-/vagrant/install/scripts/install-spdx-tools.sh
-
-# install ninka
-/vagrant/install/scripts/install-ninka.sh
-
 date > /etc/vagrant.provisioned
 
 echo "lets go!"
 cd /vagrant
 
 ./utils/fo-installdeps -e -y
 
-make CFLAGS=-I/usr/include/glib-2.0
+make all
 sudo make install
+# fix ownership
+sudo chown -R vagrant:vagrant /vagrant
```

Comment:
This modifies the permissions on the mounted folder, i.e. might cause permission problems for the user on the host system?

This is done after calling `make all`, so if there were permission problems this is already to late?